Makefile

```diff
@@ -3,57 +3,56 @@
 # Run Postgres
 .PHONY: db
 db:
-  docker start codesearch-db || docker run --name codesearch-db \
```

Comment:
If all makefiles have been switched to tabs, then you can remove the gmake requirement on macOS from the readme.

Reply:
Oh, and then you can also remove RECIPEPREFIX.

Reply:
Done.